src/ATen/function_wrapper.py

```diff
@@ -305,10 +327,43 @@ def process_option(option, output_options):
         # another function-only variant can exist without the name colliding
```

Comment:
It looks like we are following the convention of adding `[a-z]_` to handle variants for functions we are wrapping. Is this a sustainable long-term strategy? Do you imagine adding another case where we could have `m_s_q_` or w/e.

Reply:
I don't know, but it shouldn't matter in this case because it isn't publicly visible, so we can just change it later.